### ISSUE TITLE
Don't `using namespace std;` at global scope in headers

### ISSUE THREAD

**Issue by lufsc** [Lux-AI-Challenge/Lux-Design-2021#55]
While `using namespace std;` may have some benefits, it has some downsides too. Therefore the decision whether or not to use the namespace should be left to the person writing the bot. However, when you put `using namespace std;` in a header at global scope, that means that you make that decision for the user of the library, as they can't remove this statement.

The fix for this would be to move the `using namespace std;` into the `kit` namespace, so this
```cpp
using namespace std;
namespace kit {
    // ...
}
```
would become this
```cpp
namespace kit {
    using namespace std;

    // ...
}
```

This would be a breaking change though, but given that it's quite early in the competition, I think this is acceptable.

**Comment by StoneT2000** [comment]
good catch. we will merge this in in the next release and update the kits!

**Comment by StoneT2000** [comment]
to be merged in #51
